src/main/java/frc/robot/Robot.java

```diff
@@ -96,6 +125,7 @@ public void teleopPeriodic() {
      */
     @Override
     public void disabledInit() {
+        SwerveDrive.getFieldOrientedInstance().terminate();
```

Comment:
Won't work, you can't operate the motors while the robot is disabled

Reply:
It's still beneficial so when the robot is enabled again, it won't move if before being disabled it had a non-zero velocity setpoint.